src/pendulum/duration.py

```diff
@@ -238,7 +238,7 @@ def in_minutes(self) -> int:
     def in_seconds(self) -> int:
         return int(self.total_seconds())
 
-    def in_words(self, locale: str | None = None, separator: str = " ") -> str:
+def in_words(self, locale: str | None = None, separator: str = " ") -> str:
```

Comment:
```suggestion
    def in_words(self, locale: str | None = None, separator: str = " ") -> str:
```

Sorry, made a bit of a mess of this doing the changes via suggestions